### ISSUE TITLE
Docs: replace textareas with scrolling divs in code-block component

### ISSUE THREAD

**Issue by warriordog** [warriordog/mooltipage#67]
The documentation site uses textareas to handle the code blocks, but this causes issues for accessibility tech and other browser tools that assume that the textarea is for user input. This should be replace with a div for compatibility. Some things to be aware of:
- Existing textareas are sized in "rows" currently, divs will need to be sized some other way.
- These are whitespace-sensitive, so that needs to be handled in CSS or in the Mooltipage compiler. CSS is better is possible.

The code-block component is implemented in codeBlock.html.

**Comment by aaronleopold** [comment]
Hey there, I can try and take a look at this if you'd like 🙂

**Comment by warriordog** [comment]
Hello @aaronleopold, thanks for the help!
I'll go ahead and assign you this issue to you, but you're in no way obligated to complete it. Let me know if you ever change your mind.

I don't know if you've looked much at the code for the documentation website, but the example code blocks are implemented with two components: docs/src/fragments/exampleCode.html and docs/src/fragments/codeBlock.html. Most of the changes for this work will be in the `code-block` component, but code-block is only used directly by `example-code` so make sure to look there to see how it works. The entire documentation project is written using Mooltipage fragments and scripts, which aren't fully documented yet. The [HTML Usage](https://warriordog.github.io/mooltipage/sections/gettingStarted/htmlUsage.html) section might be helpful. 

If you need help understanding anything then feel free to tag me in a comment or contact me directly at [acomputerdog@gmail.com](mailto:acomputerdog@gmail.com).

**Comment by aaronleopold** [comment]
@warriordog Thanks for the clarification! I'll have time later this week to really dive in, and I'll be sure to reach out if I have any questions! 😄
